sdks/python/apache_beam/io/hdfs_integration_test/hdfs_integration_test.sh

```diff
@@ -53,6 +53,6 @@ function finally {
 }
 trap finally EXIT
 
-time docker-compose ${COMPOSE_OPT} build
+time docker-compose ${COMPOSE_OPT} build --build-arg BASE_IMAGE=$1
```

Comment:
done

Reply:
Thanks, sorry if I wasn't clear - I actually thought of adding a comment earlier  at the beginning of the script:
 https://github.com/apache/beam/blob/90a8cada932e5559e63abb2f60cd2b539bdcd89c/sdks/python/apache_beam/io/hdfs_integration_test/hdfs_integration_test.sh#L18

It could be a description of required arguments or sample usage for someone looking into how to run that script manually.